app-server/app/src/main/scala/scene/Scene.scala

```diff
@@ -130,7 +153,7 @@ object SceneService extends AkkaSystem.LoggerExecutor {
       val hasNext = (pageRequest.offset + 1) * pageRequest.limit < totalScenes // 0 indexed page offset
       val hasPrevious = pageRequest.offset > 0
       PaginatedResponse(totalScenes, hasPrevious, hasNext,
-        pageRequest.offset, pageRequest.limit, scenes)
+        pageRequest.offset, pageRequest.limit, scenes.toSeq)
```

Comment:
Oh, just remembered - the reason it is an iterable is because the group by happening on the server side.